### DISCUSSION TITLE
little question

### DISCUSSION THREAD

**Discussion by guiworldtv2** [bitsbb01/m3u8_creator#6] [Q&A]
I took a quick look at the codes and they seem to be very interesting, I want to merge some m3u8 lists that I have into one, I don't understand much about programming, could you help me to make this system? apparently somewhere in this repository this has already been created, I'm lost in the midst of so many codes and commands hugs

**Comment by danfercf1** [comment]
> I took a quick look at the codes and they seem to be very interesting, I want to merge some m3u8 lists that I have into one, I don't understand much about programming, could you help me to make this system? apparently somewhere in this repository this has already been created, I'm lost in the midst of so many codes and commands hugs

I'm working in a m3u generator, it merges a bunch of m3u files and its epgs lists also, if you want, you can contact me

**Reply by bitsbb01** [reply]
hi, sorry for the late reply, i've had brain surgery and was in hospital... 
so do excuse me if it seems a bit haywire me explaining it... lol

so if you goto :- [m3u8_creator](https://github.com/bitsbb01/m3u8_creator)/[.github](https://github.com/bitsbb01/m3u8_creator/tree/main/.github)/[workflows](https://github.com/bitsbb01/m3u8_creator/tree/main/.github/workflows)/m3u8_converter.yml

you'll see i've used 

        echo "Command: './scripts/download_file_allow_failiure.sh "download.m3u8" "${M3U8_STATIC_DIR}" "download.m3u8"'"
          ./scripts/download_file_allow_failiure.sh "download.m3u8" "${M3U8_STATIC_DIR}" "download.m3u8"

this will allowing my shell script to run and grab the m3u8 file and then dump it into my static folder ready for combining into the aio.

i do have other githubs that are running, for example to get the m3u8's from cat nightly so this main github is able to pull from it..

https://raw.githubusercontent.com/bitsbb01/m3u8_creator/main/.github/workflows/m3u8_converter.yml

if you cant see the main cron (m3u8_converter.yml) running let me know.
